test/build/html-environment-variables/public/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en" data-build-time="%SNOWPACK_PUBLIC_BUILD_TIME%" data-undefined="%SNOWPACK_PUBLIC_BUILD_UNDEFINED%">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="Web site created using create-snowpack-app" />
+    <link rel="stylesheet" type="text/css" href="/index.css" />
+    <title>Snowpack App</title>
```

Comment:
Maybe also add a `%SNOWPACK_PUBLIC_APP_NAME%` here for the test.

Reply:
I test one variable that is set (`SNOWPACK_PUBLIC_BUILD_TIME`) and one that is not set (`SNOWPACK_PUBLIC_BUILD_UNDEFINED`).

Would adding `SNOWPACK_PUBLIC_APP_NAME` add a use case that is not yet covered? I'm trying to keep the tests as concise as possible

Reply:
No, I guess one positive and negative is good enough.